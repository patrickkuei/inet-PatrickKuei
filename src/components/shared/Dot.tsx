export default function Dot() {
  return <span className="text-gray-400">â€¢</span>
}
