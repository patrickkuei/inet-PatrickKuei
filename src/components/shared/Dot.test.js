import renderer from 'react-test-renderer'
import Dot from './Dot'

it('render Dot', () => {
  const component = renderer.create(<Dot />)

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
