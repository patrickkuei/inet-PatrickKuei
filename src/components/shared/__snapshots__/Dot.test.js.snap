// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Dot 1`] = `
<span
  className="text-gray-400"
>
  â€¢
</span>
`;
